src/components/Facility/DoctorVideoSlideover.tsx

```diff
@@ -238,6 +240,25 @@ function UserListItem({ user }: { user: UserAnnotatedWithGroup }) {
     }
   }
 
+  const { t } = useTranslation();
+  const [copied, setCopied] = useState(false);
+  const handleCopy = async (
+    e: React.MouseEvent<HTMLAnchorElement, MouseEvent>,
+  ) => {
+    e.stopPropagation();
+    await navigator.clipboard.writeText(user?.alt_phone_number || "");
+
+    Notification.Success({
+      msg: t("phone_number_copied"),
+    });
+
+    setCopied(true);
+
+    setTimeout(() => {
+      setCopied(false);
+    }, 2000);
+  };
```

Comment:
can we make a utility function called `copyToClipboard` in utils.ts and invoke that instead?

Reply:
yeah sure @rithviknishad , 
but can we go with a custom hook instead of util, we also need state of the copy to change icon, incase we don't want to include that extra logic in 'DoctorVideoSlideover'
The same logic can be reused at other places like 'Warranty Card' etc.

Reply:
> can we make a utility function called `copyToClipboard` in utils.ts and invoke that instead?

yeah sure @rithviknishad ,
but can we go with a custom hook instead of util, we also need state of the copy to change icon, incase we don't want to include that extra logic in 'DoctorVideoSlideover'
The same logic can be reused at other places like 'Warranty Card' etc.